api/src/main/java/org/pmiops/workbench/user/DevUserRegistrationService.java

```diff
@@ -0,0 +1,8 @@
+package org.pmiops.workbench.user;
+
+import com.google.api.services.oauth2.model.Userinfoplus;
+import org.pmiops.workbench.db.model.DbUser;
+
+public interface DevUserRegistrationService {
```

Comment:
nit: not sure if "dev" is an approved abbrev.

Reply:
I think dev is pretty well-established usage – e.g. `project.rb dev-up`